app/views/organisations/_organisations_list.html.erb

```diff
@@ -33,31 +33,40 @@
             },
           } do %>
             <% if @presented_organisations.ministerial_organisation?(organisation_type) %>
-              <%= render "govuk_publishing_components/components/organisation_logo", {
-                organisation: {
-                  name: organisation["logo"]["formatted_title"],
-                  url: organisation["href"],
-                  brand: organisation["brand"],
-                  crest: organisation["logo"]["crest"]
-                },
-                heading_level: 3,
-              } %>
+              <div class="govuk-!-margin-bottom-3">
+                <%= render "govuk_publishing_components/components/organisation_logo", {
+                  organisation: {
+                    name: organisation["logo"]["formatted_title"],
+                    url: organisation["href"],
+                    brand: organisation["brand"],
+                    crest: organisation["logo"]["crest"]
+                  },
+                  heading_level: 3,
+                } %>
+              </div>
             <% else %>
-              <%= link_to(organisation["title"], organisation["href"], class: "govuk-link organisation-list__item-title") %>
+              <div class="govuk-!-margin-bottom-3">
+                <h3 class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-bottom-0 govuk-!-display-inline govuk-!-margin-right-2">
```

Comment:
Something's not quite right here, this doesn't look like heading

![image](https://user-images.githubusercontent.com/71266765/152957769-05599231-581c-40f1-bab6-30675303a735.png)

Reply:
Hmm, you make a good point. Initially I thought it made sense as it preserves the hierarchy of the siblings that are sort of parents, but I'm now considering whether any of these h3s should actually be headings. The relationship is that the listed org works with the ones listed under it, but it's not a parent to them. I think we should probably be removing all the heading levels throughout.

Reply:
Ah that is a good point. I'd changed the markup to use a heading because the document outline is currently broken for entries that don't use the organisation logo component but do contain 'works with' content.

![validator w3 org_nu__doc=https%3A%2F%2Fwww gov uk%2Fgovernment%2Forganisations showoutline=yes (1)](https://user-images.githubusercontent.com/87758239/152959645-76d259ec-b8e8-4fcc-bc35-10559d936c24.png)

As Max mentioned above, this should probably be tackled separately so will create a new card and revert this back to the previous markup.